pkgs/data/sgml+xml/schemas/sgml-dtd/docbook/4.2.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+let
+
+  src = fetchurl {
+    url = "http://www.oasis-open.org/docbook/sgml/4.2/docbook-4.2.zip";
```

Comment:
The version number should be substituted.

Reply:
This is fixed version of an ancient standard, there will be no updates.